script/int-log-compiler.sh

```diff
@@ -253,16 +266,52 @@ env TPM20TEST_TCTI_NAME="socket" \
 ret=$?
 echo "Script returned $ret"
 
+#We check the state before a reboot to see if transients and NV were chagned.
 env TPM20TEST_TCTI_NAME="socket" \
     TPM20TEST_SOCKET_ADDRESS="127.0.0.1" \
     TPM20TEST_SOCKET_PORT="${SIM_PORT_DATA}" \
-    G_MESSAGES_DEBUG=all ./test/helper/tpm_transientempty
+    G_MESSAGES_DEBUG=all ./test/helper/tpm_dumpstate>$TPMSTATE_FILE2
 if [ $? -ne 0 ]; then
-    echo "TPM transient area not empty or generally failed after test"
+    echo "Error during dumpstate"
     ret=99
     break
 fi
 
+if [ "$(cat $TPMSTATE_FILE1)" != "$(cat $TPMSTATE_FILE2)" ]; then
```

Comment:
Not something that prevents merging it, but would using diff be more appropriate here?
if [ "$(diff $TPMSTATE_FILE1 $TPMSTATE_FILE2 -ne 0) ];
Probably something to improve.

Reply:
Honestly, I tried it, but for some reason did not work, so I went the lazy route known to work... ;-)